cms/static/js/views/course_info_update.js

```diff
@@ -1,8 +1,9 @@
 define(['js/views/validation', 'codemirror', 'js/models/course_update',
         'common/js/components/views/feedback_prompt', 'common/js/components/views/feedback_notification',
-        'js/views/course_info_helper', 'js/utils/modal', 'js/utils/date_utils'],
+        'js/views/course_info_helper', 'js/utils/modal', 'js/utils/date_utils',
+        'edx-ui-toolkit/js/utils/html-utils'],
     function(ValidatingView, CodeMirror, CourseUpdateModel, PromptView, NotificationView,
-        CourseInfoHelper, ModalUtils, DateUtils) {
+        CourseInfoHelper, ModalUtils, DateUtils, HtmlUtils) {
```

Comment:
Much better! I'd probably move CodeMirror first as a vendor library, and then I'd move CourseUpdateModel towards the end as it is a model for this code itself.